public_html/wp-content/themes/campsite-2017/inc/jetpack.php

```diff
@@ -27,6 +28,19 @@ function jetpack_setup() {
 		)
 	);
 
+	$featured_image_enabled = ! ( function_exists( 'wcorg_skip_feature' ) && wcorg_skip_feature( 'cs17_display_featured_image' ) );
+	add_theme_support( 'jetpack-content-options', array(
+		'featured-images'    => array(
+			'archive'           => true,
+			'archive-default'   => false,
```

Comment:
Why not apply the `$featured_image_enabled` variable here as well?

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/111/commits/e539b28421d0f44e708b3fd945c0bf76df97b08b